src/main/resources/api/data-repository-openapi.yaml

```
@@ -4066,6 +4066,70 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  ##############################################################################
+  ## SNAPSHOT BUILDER API ENDPOINTS
+  ## ⚠️ NOTE: ALL SNAPSHOT APIS ARE SUBJECT TO CHANGE
+  ##############################################################################
+  /api/repository/v1/datasets/{id}/snapshot-builder/settings:
```

It looks like the existing convention for paths like this is to use camel case instead of dashes. For example, I see `/api/repository/v1/snapshots/{id}/unlinkDuosDataset`, `/api/repository/v1/snapshots/roleMap`, `/api/repository/v1/duos/syncAuthorizedUsers`.
```suggestion
  /api/repository/v1/datasets/{id}/snapshotBuilder/settings:
```